wgengine/magicsock/magicsock.go

```diff
@@ -2822,17 +2808,7 @@ func (c *Conn) ReSTUN(why string) {
 }
 
 func (c *Conn) initialBind() error {
```

Comment:
this initialBind is probably not worth keeping at this point? just move the rebind to the sole caller?